schema/project-level/project-schema.json

```
@@ -1426,6 +1435,32 @@
       "required": [
         "id"
       ]
+    },
+    "Person": {
+      "title": "Person",
+      "description": "A natural person.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "Identifier",
+          "description": "A local identifier for this person within the people array.",
+          "type": "string",
+          "minLength": 1
+        },
+        "name": {
+          "title": "Name",
+          "description": "The full name of this person.",
```

To match construction of `jobTitle`:

```suggestion
          "description": "The full name of the person.",
```